src/components/v1/V1Project/ProjectActivity/index.tsx

```diff
@@ -306,6 +299,9 @@ export default function ProjectActivity() {
             type="text"
             icon={<DownloadOutlined />}
             onClick={() => setDownloadModalVisible(true)}
+            style={{
+              display: count > 0 ? 'inline-block' : 'none',
+            }}
```

Comment:
**question** curious on the advantage of using styling vs conditional rendering. Thoughts?

Reply:
lol just a brain fart. fixed.